src/main/java/com/google/gcs/sdrs/controller/EventsController.java

```diff
@@ -102,13 +103,27 @@ private void validateExecutionEvent(ExecutionEventRequest request) throws Valida
     } else {
       switch (request.getExecutionEventType()) {
         case USER_COMMANDED:
+          ValidationResult validateTargetNameResult =
+              FieldValidations.validateFieldFollowsBucketNamingStructure(
```

Comment:
The case logic is getting a bit verbose, consider refactoring into its own helper method.

Reply:
done